src/EditorFeatures/Core/Remote/SolutionChecksumUpdater.cs

```diff
@@ -22,7 +21,12 @@ namespace Microsoft.CodeAnalysis.Remote
     internal sealed class SolutionChecksumUpdater
     {
         private readonly Workspace _workspace;
-        private readonly IGlobalOperationNotificationService _globalOperationService;
+
+        /// <summary>
+        /// We're not at a layer where we are guaranteed to have an IGlobalOperationNotificationService.  So allow for
+        /// it being null.
+        /// </summary>
+        private readonly IGlobalOperationNotificationService? _globalOperationService;
```

Comment:
IGlobalOperationNotificationService has been made into a normal mef service (no longer a workspace service).  It is not exported from the core layers.  Instead, there is an export at teh VS, Remote, and Test layers.